packages/mobile/src/send/SendSearchInput.tsx

```diff
@@ -1,64 +1,35 @@
-import TextInput, { TextInputProps } from '@celo/react-components/components/TextInput'
-import withTextInputLabeling from '@celo/react-components/components/WithTextInputLabeling'
-import withTextInputPasteAware from '@celo/react-components/components/WithTextInputPasteAware'
-import colors from '@celo/react-components/styles/colors'
-import { isValidAddress } from '@celo/utils/src/address'
+import SearchInput from '@celo/react-components/components/SearchInput'
 import * as React from 'react'
 import { useTranslation } from 'react-i18next'
 import { StyleSheet, View } from 'react-native'
 import { Namespaces } from 'src/i18n'
-import Search from 'src/icons/Search'
-
-const RecipientSearchInput = withTextInputPasteAware(
-  withTextInputLabeling<TextInputProps>(TextInput),
-  { right: 22 }
-)
 
 interface SendSearchInputProps {
-  isPhoneEnabled: boolean
   onChangeText: (value: string) => void
 }
 
 // Input field for Send screen
 export function SendSearchInput(props: SendSearchInputProps) {
+  function handleChangeText(value: string) {
+    setInput(value)
+    onChangeText(value)
+  }
+
   const { t } = useTranslation(Namespaces.sendFlow7)
-  const { isPhoneEnabled, onChangeText } = props
+  const { onChangeText } = props
   const [input, setInput] = React.useState('')
 
-  const handleChangeText = React.useCallback(
-    (value: string) => {
-      setInput(value)
-      onChangeText(value)
-    },
-    [setInput]
-  )
-
   return (
     <View style={styles.textInputContainer}>
-      <RecipientSearchInput
-        placeholder={isPhoneEnabled ? t('nameOrPhoneNumber') : t('walletAddress')}
-        value={input}
-        onChangeText={handleChangeText}
-        icon={isPhoneEnabled ? <Search /> : undefined}
-        title={isPhoneEnabled ? undefined : t('global:to')}
-        style={styles.textInput}
-        shouldShowClipboard={isValidAddress}
-        testID={'RecipientSearchInput'}
-      />
+      <SearchInput placeholder={t('global:search')} value={input} onChangeText={handleChangeText} />
```

Comment:
Does this new input still show the paste button when there's an address in the clipboard?

Reply:
Well, I do not see it in the design, @coreycelo can you confirm that search field at `Send` screen does not have paste functionality?

Reply:
Happy to include it, it's a nice Easter egg. As long as we allow the normal text paste (tap and hold to reveal the option), I don't see this as a blocker. I'll mock it up.

Reply:
@coreycelo  Is it ok we ship it now without this functionality?

Reply:
Sounds good to me